onnxruntime/core/providers/xnnpack/nn/conv.cc

```diff
@@ -102,8 +101,13 @@ Status Conv::Compute(OpKernelContext* context) const {
     reshape_fn = xnn_reshape_convolution2d_nhwc_qu8;
   } else if (conv_type_ == OpComputeType::op_compute_type_qs8_per_channel) {
     reshape_fn = xnn_reshape_convolution2d_nhwc_qs8_qc8w;
+  } else if (conv_type_ == OpComputeType::op_compute_type_fp16) {
+    reshape_fn = xnn_reshape_convolution2d_nhwc_f16;
   }
 
+  if (!op0_.get()) {
+    throw std::invalid_argument("op0 ------");
```

Comment:
nit: ORT_THROW, and please add a message that will be meaningful to someone reading a log.

Reply:
Sorry, it's a debug info. I'll remove it.